### ISSUE TITLE
Consider CoinGecko API

### ISSUE THREAD

**Issue by tmlee** [chnsh/crypto-index-fund#1]
Cofounder at CoinGecko.com here, may I suggest using our API https://www.coingecko.com/api instead of the need to use scrappy to crawl our site for data

**Comment by chnsh** [comment]
Yes, that's the first To-do. When I first made this, you guys did not have an API, and I had to resort to scraping. 

By the way, coingecko is awesome!

**Comment by tmlee** [comment]
Thanks! Its finally here! :shipit: 
Do let us know if there is any problem you encounter while using the API and how we can help

**Comment by chnsh** [comment]
Definitely. 

I'll implement it as soon as I find some time. I had this in private repositories since December. Finally, a Sunday helped me open source it. I'll let you know once it's done!

**Comment by chnsh** [comment]
I glanced at the API, while it gives out comprehensive information (much more than what I get from the scraper), I could not find the summary stats (like liquidity, developer score, etc)

Do I need to compute the summary stats from the raw information? Also, is it possible to add it to the API?
